src/components/cc-block-cli/cc-block-cli.js

```diff
@@ -0,0 +1,122 @@
+import { css, html, LitElement } from 'lit';
+import { accessibilityStyles } from '../../styles/accessibility.js';
+import { iconRemixArrowUpSLine as iconArrowUp } from '../../assets/cc-remix.icons.js';
+import '../cc-icon/cc-icon.js';
+
+/**
+ * @typedef {import('./cc-block-cli.types.js').BlockToggleState} BlockToggleState
+ */
+
+/**
+ * A display component with mostly HTML+CSS and an open/close toggle feature organized with slots to display information such as CLI commands.
+ *
+ * @cssdisplay block
+ *
+ * @slot tab-title - The title of the tab. Try to only use text.
+ * @slot link - A zone dedicated to link, for example to documentation.
+ * @slot content - A zone dedicated to main content.
+ */
+
+export class CcBlockCli extends LitElement {
```

Comment:
fix: Since this component seams to be very generic, I'm not sure we should name it with `cli` suffix. Suggestion:

* `CcBlockDetail`
* `CcBlockFooter`